custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -893,12 +875,9 @@ def _prepare_docker_command(
         in_docker_fit_row_weights_filename = "/opt/fit_row_weights.csv"
         in_docker_runtime_parameters_file = "/opt/runtime_parameters.yaml"
 
-        security_context = security_context or SecurityContext(
-            user_id=os.getuid(), group_id=os.getgid()
-        )
         docker_cmd = (
-            "docker run --rm --init --entrypoint '' --interactive"
-            f" --user {security_context.user_id}:{security_context.group_id}"
+            f"docker run --rm --init --entrypoint '' --interactive"
+            f" --user {os.getuid()}:{os.getgid()}"
```

Comment:
Are these changes made for running locally? Wondering why is this in scope of the PR

Reply:
Just rolled it back to the original implementation. This is somehow imitates what we do in DR backend - an arbitrary user.